src/condor_daemon_core.V6/condor_daemon_core.h

```diff
@@ -951,6 +952,24 @@ class DaemonCore : public Service
                          HandlerType          handler_type = HANDLE_READ,
                          void **              prev_entry = NULL);
 
+    int Register_Socket (Stream*           iosock,
+                         const char *      iosock_descrip,
+                         StdSocketHandler  handler,
+                         const char *      handler_descrip,
+                         HandlerType       handler_type = HANDLE_READ,
+                         void **           prev_entry = NULL) {
+        return Register_Socket (iosock,
+                                iosock_descrip,
+                                (SocketHandler)nullptr,
+                                (SocketHandlercpp)nullptr,
+                                handler_descrip,
+                                nullptr, // Service
+                                handler_type,
+								false,
+                                prev_entry,
+								&handler);
```

Comment:
If you happen to make another commit on this PR, it would bring joy to my hear to see the indentation here become consistent.